### ISSUE TITLE
Error on Depend attrubute

### ISSUE THREAD

**Issue by girishcp** [buttflattery/yii2-formwizard#80]
@buttflattery 

I have updated the latest version1.7.9 and i am getting the following error while using this one of my fields in the STEP_TYPE_TABULAR.

Setting unknown property: yii\bootstrap\ActiveField::depends

I have also tried with V1.6.17 and it doesn't show the error but it was not hiding the depend fields in the STEP_TYPE_TABULAR. 

Please check this issue.

**Comment by buttflattery** [comment]
hmmm, looks like another part missing @girishcp  sorry about that will be fixed soon

**Comment by girishcp** [comment]
@buttflattery  
did you get a chance to look into this issue?

**Comment by buttflattery** [comment]
yeah i am working on both the fixes and will push fix for both of them soon @girishcp

**Comment by girishcp** [comment]
@buttflattery Thanks

**Comment by girishcp** [comment]
@buttflattery 

I hope you are doing well.

Just point you one more issue, while you are working on the Depend attribute issue. Please check also applying to depend attribute in the multiple fields to the same tabular STEP because I am getting an error while applying the same.

_**Uncaught SyntaxError: redeclaration of let thenCallback_0**_

when I remove my second depend on an attribute from one of the fields then it starts working.

Thanks for your support

**Comment by buttflattery** [comment]
@girishcp alright i will look into it too this looks familiar and i think i had fixed it in the past too , dont know why is it ocurring for you again, will check it too

**Comment by girishcp** [comment]
@buttflattery Any chance to resolve the "Depend" attribute issue.

**Comment by girishcp** [comment]
@buttflattery 
I hope you are doing well!
Can you please tell me, if you get a chance to update the wizard, actually I have a release in the coming days. Thank you

**Comment by girishcp** [comment]
@buttflattery 
I was just debugging the issue, when we click the "Add" button in the tabular form then it's adding the form twice and it's because of using the value "'toolbarPosition' => 'both',", if we set "bottom" or "top" as value then it works fine.

**Comment by buttflattery** [comment]
@girishcp yeah i have noticed that too , actually the change for the support of the default options wasnt that straight forward as I thought it would be and I am still figuring out the proper way to implement it and that is why havent pushed any more changes, sorry for the delay you had to face. Will be done by this weekend

**Comment by girishcp** [comment]
@ZulfikriLbs Did you get time to check this issue as well?
